classes/class-wpcom-liveblog-entry-query.php

```diff
@@ -19,7 +19,7 @@ public function __construct( $post_id, $key ) {
 	 * @param array $args the same args for the core `get_comments()`.
 	 * @return array array of `WPCOM_Liveblog_Entry` objects with the found entries
 	 */
-	private function get( $args = array() ) {
+	public function get( $args = array() ) {
```

Comment:
This has been changed so it can be used when we grab all of the key events for the new key events API. You can find it in `class-wpcom-liveblog-entry-key-events` on line 394.